core/domain/exp_domain.py

```diff
@@ -3291,6 +3319,11 @@ def _migrate_to_latest_yaml_version(
                 exploration_dict)
             exploration_schema_version = 17
 
+        if exploration_schema_version == 17:
```

Comment:
I think some things are missing. The "current exp schema version" constant should be updated and I would expect some changes to the test file...

(Please cross-check against a previous PR that did an exploration migration to make sure all the bits and pieces are accounted for.)

Reply:
done